obspy/taup/__init__.py

```diff
@@ -140,50 +140,106 @@
+>>> arrivals.plot_rays()  # doctest: +SKIP
 
 .. plot::
     :width: 50%
     :align: center
 
     from obspy.taup import TauPyModel
-    TauPyModel().get_ray_paths(500, 130).plot()
+    TauPyModel().get_ray_paths(500, 130, phase_list=["ttbasic"]).plot_rays()
 
 Plotting will only show the requested phases:
 
 >>> arrivals = model.get_ray_paths(
 ...     source_depth_in_km=500,
 ...     distance_in_degree=130,
 ...     phase_list=["Pdiff", "Sdiff", "pPdiff", "sSdiff"])
->>> arrivals.plot()  # doctest: +SKIP
+>>> ax = arrivals.plot_rays()  # doctest: +SKIP
 
 .. plot::
     :width: 50%
     :align: center
 
     from obspy.taup import TauPyModel
-    TauPyModel().get_ray_paths(
-        500, 130,
-        phase_list=["Pdiff", "Sdiff", "pPdiff", "sSdiff"]).plot()
+    TauPyModel().get_ray_paths(500, 130,
+        phase_list=["Pdiff", "Sdiff", "pPdiff", "sSdiff"]).plot_rays()
 
 Additionally, Cartesian coordinates may be used instead of a polar grid:
 
 >>> arrivals = model.get_ray_paths(source_depth_in_km=500,
 ...                                distance_in_degree=130,
 ...                                phase_list=["ttbasic"])
->>> arrivals.plot(plot_type="cartesian")  # doctest: +SKIP
+>>> ax = arrivals.plot_rays(plot_type="cartesian")  # doctest: +SKIP
 
 .. plot::
     :width: 75%
     :align: center
 
     from obspy.taup import TauPyModel
     TauPyModel().get_ray_paths(
-        500, 130, phase_list=["ttbasic"]).plot(plot_type="cartesian")
+        500, 130, phase_list=["ttbasic"]).plot_rays(plot_type="cartesian")
+
+Travel times for these ray paths can be plotted using the
+:meth:`~obspy.taup.tau.Arrivals.plot_times` method:
+
+>>> arrivals = model.get_ray_paths(source_depth_in_km=500,
+...                                distance_in_degree=130)
+>>> ax = arrivals.plot_times()  # doctest: +SKIP
+
+.. plot::
+    :width: 50%
+    :align: center
+
+    from obspy.taup import TauPyModel
+    ax = TauPyModel().get_ray_paths(500, 130).plot_times()
+
+Alternatively, convenience wrapper functions plot the arrival times
+and the ray paths for a range of epicentral distances.
+
+The travel times wrapper function is:
+
+>>> from obspy.taup import plot_travel_times
+>>> import matplotlib.pyplot as plt
+>>> fig, ax = plt.subplots()
+>>> ax = plot_travel_times(source_depth=10, phase_list=["P","S","PP"],
+... ax=ax, fig=fig)
+There were 2 epicentral distances without an arrival
+
+.. plot::
+    :width: 50%
+    :align: center
+
+    from obspy.taup import plot_travel_times
+    import matplotlib.pyplot as plt
+
+    fig, ax = plt.subplots()
+    ax = plot_travel_times(source_depth=10, ax=ax, phase_list=["P","S","PP"],
+    ... fig=fig)
+
+The ray path plot wrapper function is:
+
+>>> from obspy.taup import plot_ray_paths
+>>> import matplotlib.pyplot as plt
+>>> ax = plt.subplot(111, polar=True)
+>>> fig = ax.figure
+>>> ax = plot_ray_paths(source_depth=100, ax=ax, fig=fig)
+There were rays for all but the following epicentral distances:
```

Comment:
See above, please add an empty line below the code block, I think that's nicer to look at in an editor.

Reply:
I didn't think that was possible; the doc string needs to match the output of the code block. An empty line would throw a wrench into that idea. But I must be missing something.

Reply:
Blank lines that are really in the output are specified as `<BLANKLINE>` in doctests. So adding a blank line for nicer look is totally safe. See http://www.sphinx-doc.org/en/stable/ext/doctest.html?highlight=doctest#directive-doctest